packages/modelviewer.dev/examples/scenegraph/index.html

```diff
@@ -16,614 +16,578 @@
 -->
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <title>&lt;model-viewer&gt; Scene Graph Worklet</title>
   <meta charset="utf-8">
   <meta name="description" content="&lt;model-viewer&gt; scene graph examples">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link type="text/css" href="../../styles/examples.css" rel="stylesheet" />
   <link type="text/css" href="../../styles/docs.css" rel="stylesheet" />
-  <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
-
+  <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png" />
   <!-- 💁 OPTIONAL: The :focus-visible polyfill removes the focus ring for some input types -->
   <script src="../../node_modules/focus-visible/dist/focus-visible.js" defer></script>
-
   <script>
-    window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
+    window.ga = window.ga || function () { (ga.q = ga.q || []).push(arguments) }; ga.l = +new Date;
```

Comment:
I'm guessing you had auto-format set for this file. Can you revert it so our formatting stays? I know it's not great, but it's the only way we've found to keep the example text from having a bunch of tabs in front of every line. Also, I can't really see your changes when the diff is this bad.

Reply:
I noticed this also, reverted on next commit